.github/workflows/ci.yml

```diff
@@ -19,18 +19,18 @@ env:
 jobs:
   matrix-setup:
     name: Matrix Setup
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Do you happen to have a log of the failure?

Reply:
`Build dependency: Please install Python 2.x`

https://github.com/Habbie/openwrt-vlmcsd/actions/runs/5243824268/jobs/9469004588

Reply:
Can probably fix it by installing Python 2.x, but I don't really want to do that, and I did not dig any further yet.

Reply:
addressed by refreshing `matrix.json`, which got rid of 19.07